linux/LinuxProcessTable.c

```diff
@@ -1639,10 +1645,38 @@ static bool LinuxProcessTable_recurseProcTree(LinuxProcessTable* this, openat_ar
          }
       }
 
-      /*
-       * Section gathering non-critical information that is independent from
-       * each other.
-       */
+      /* Reset, causes processes might enter namespaces at runtime. */
```

Comment:
According to [pid_namespaces(7)](https://www.man7.org/linux/man-pages/man7/pid_namespaces.7.html):

> To put things another way: a process's PID namespace membership
       is determined when the process is created and cannot be changed
       thereafter.

<details>

> Calls to [setns(2)](https://www.man7.org/linux/man-pages/man2/setns.2.html) that specify a PID namespace file descriptor
       and calls to [unshare(2)](https://www.man7.org/linux/man-pages/man2/unshare.2.html) with the CLONE_NEWPID flag cause children
       subsequently created by the caller to be placed in a different
       PID namespace from the caller.  (Since Linux 4.12, that PID
       namespace is shown via the /proc/pid/ns/pid_for_children file, as
       described in [namespaces(7)](https://www.man7.org/linux/man-pages/man7/namespaces.7.html).)  
> These calls do not, however, change
       the PID namespace of the calling process, because doing so would
       change the caller's idea of its own PID (as reported by
       getpid()), which would break many applications and libraries.


</details>

Reply:
Thanks, fixed.